demos/demos.scss

```diff
@@ -14,16 +14,17 @@
   limitations under the License
 */
 
```

Comment:
I feel like you can just put one `/* stylelint-disable */` at the top level here.